test/e2e/network/endpointslice.go

```diff
@@ -524,6 +526,209 @@ var _ = common.SIGDescribe("EndpointSlice", func() {
 		framework.ExpectNoError(err)
 		framework.ExpectEqual(len(epsList.Items), 0, "filtered list should have 0 items")
 	})
+
+	ginkgo.It("should forward traffic to a Service with two custom EndpointSlices with duplicate addresses", func() {
```

Comment:
This doesn't describe the test well (or differentiate it from the other one). Maybe
`"should support a Service with multiple ports specified in multiple EndpointSlices"`?

The fact that the slices are "custom" is not really relevant. In theory the EndpointSlice controller could have created them that way.

Also, FTR, I'm not certain that this behavior is actually expected. I would have said that it was erroneous to have multiple EndpointSlices for the same service with different Ports. (Though, I guess, what's the point of having the Ports in the EndpointSlice if they can't be different from the Ports in the Service?)

Reply:
this is a consequence of this https://github.com/kubernetes/kubernetes/pull/114155 , basically, if you create an Endpoint with multiple subsets, the mirroring controller creates this weird Slices
I thought it was a simpler problem, at this point I should open an issue and explain it correctly

Reply:
done